collections/course_records.ts

```diff
@@ -55,6 +55,9 @@ if(Meteor.isServer) {
       data: {
         type: Object,
         optional: true
+      },
+      attempted: {
```

Comment:
We already talked about this: I want an array of date objects indicating when the user attempted the task as opposed to just a number.